commands/azureCommands/task-utils/showTaskManager.ts

```diff
@@ -0,0 +1,37 @@
+import * as vscode from 'vscode';
+
+export class TaskContentProvider implements vscode.TextDocumentContentProvider {
+    public static scheme: string = 'task';
+    private onDidChangeEvent: vscode.EventEmitter<vscode.Uri> = new vscode.EventEmitter<vscode.Uri>();
+
+    constructor() { }
+
+    public provideTextDocumentContent(uri: vscode.Uri): string {
+        return decodeBase64(JSON.parse(uri.query).content);
+    }
+
+    get onDidChange(): vscode.Event<vscode.Uri> {
+        return this.onDidChangeEvent.event;
+    }
+
+    public update(uri: vscode.Uri, message: string): void {
+        this.onDidChangeEvent.fire(uri);
+    }
+}
+
+export function decodeBase64(str: string): string {
+    return Buffer.from(str, 'base64').toString('ascii');
```

Comment:
Does it support unicode? If yes, should you consider `toString('utf8')`?

Reply:
Changed it to that